### PR TITLE
Update members object

### PR DESCRIPTION
**Pull request by sumitd94**

### CODE REVIEW

**Review by swarajpure — COMMENTED** [review]
What if we just directly link the `User` object from `/users` API contract? Otherwise everytime we make a change in the `user` object, we would have to update this API contract along with the users API contract. What do you say @sumitd94? 🤔

**Comment by sumitd94** [comment]
> What if we just directly link the `User` object from `/users` API contract? Otherwise everytime we make a change in the `user` object, we would have to update this API contract along with the users API contract. What do you say @sumitd94? 🤔

Does this API return everything that is mentioned in the user object? The reason why I added these two properties is because the API is returning these and it's not mentioned here.

**Comment by swarajpure** [comment]
> 
> 
> > What if we just directly link the `User` object from `/users` API contract? Otherwise everytime we make a change in the `user` object, we would have to update this API contract along with the users API contract. What do you say @sumitd94? 🤔
> 
> Does this API return everything that is mentioned in the user object? The reason why I added these two properties is because the API is returning these and it's not mentioned here.

Yes, it returns the user object only. The tokens, email and phone are not being sent in any of these routes (mentioned so in the users API contract)

**Comment by sumitd94** [comment]
> > > What if we just directly link the `User` object from `/users` API contract? Otherwise everytime we make a change in the `user` object, we would have to update this API contract along with the users API contract. What do you say @sumitd94? 🤔
> > 
> > 
> > Does this API return everything that is mentioned in the user object? The reason why I added these two properties is because the API is returning these and it's not mentioned here.
> 
> Yes, it returns the user object only. The tokens, email and phone are not being sent in any of these routes (mentioned so in the users API contract)

Ok then, let's do it then.

File: members/README.md [hunk 1]
```diff
@@ -2,25 +2,7 @@
...
-  'linkedin_id': string,
-  'twitter_id': string,
-  'instagram_id': string,
-  'site': string,
-  'github_display_name': string,
-  'isMember': boolean,
-  'tokens': {}
-}
-```
+same as the [user's](../users) object
```
Comment (swarajpure) [inline members/README.md:5]: Can we link the user object instead? `https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#user-object` 

Can we capitalize `same`? Also, better to call it `user object` than `user's object`?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Some minor changes

File: members/README.md (hunk 1 above)
Comment (sumitd94) [reply members/README.md]: https://github.com/Real-Dev-Squad/website-api-contracts/pull/32/commits/addc5eca0c0937713c79bfc5751950d3e66f6c17

**Review by swarajpure — DISMISSED** [review]
Looks good! Thanks for the quick fix ⚡

**Review by swarajpure — APPROVED** [review]

**Pull request merged**